coursebuilder/modules/explorer/_static/components/pages/certification.html

```diff
@@ -30,56 +38,48 @@ <h1 class="intro-title">Get Certified by Google</h1>
       <!--  Awards  -->
       <div class="section awards">
         <div class="carousel-section-wrapper">
-          <h2 class="section-label">An award winning platform</h2>
+          <h2 class="section-label">Partners</h2>
           <div class="owl-carousel owl-theme awards-carousel">
             <div class="item">
-              <img src="/modules/explorer/_static/images/award2.png" alt="">
+              <img src="/modules/explorer/_static/images/hha.gif" alt="">
```

Comment:
@andela-iakande should be a link to the partner's site, alt should be partner's name.